recipes/cryptominisat/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "cryptominisat" %}
+{% set version = "5.0.1" %}
+{% set sha256 = "789d53c90ecbd5910f48e4e1bd835e374b268e0b3872a81fd5c6178a30a5e263" %}
+{% set author = "msoos" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/{{ author }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+about:
+  home: https://github.com/msoos/cryptominisat
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  summary: An advanced SAT Solver https://www.msoos.org
+  description: An advanced SAT Solver https://www.msoos.org
```

Comment:
You need to add yourself as a maintainer:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L70-L75